.envrc.template

```diff
@@ -4,35 +4,35 @@
 ## * These are the values you'll need to be prepared to update to run Skate locally
 
 ## URL of the [MBTA v3 API](https://github.com/mbta/api) for retrieving live train positions
-export API_URL=
+# export API_URL=
 
 ## Access key for the MBTA v3 API
 ## * Request your personal API key from [MBTA Realtime API](https://api-v3.mbta.com/)
-export API_KEY=
+# export API_KEY=
 
 ## Postgres configuration: username, password, and hostname
 ## * Your local Postgres server should go here
-export POSTGRES_USERNAME=
-export POSTGRES_PASSWORD=
-export POSTGRES_HOSTNAME=
+# export POSTGRES_USERNAME=
+# export POSTGRES_PASSWORD=
+# export POSTGRES_HOSTNAME=
 
 ## API key from [Swiftly Transitime API](https://swiftly-inc.stoplight.io/docs/realtime-standalone/YXBpOjI4NDM2MDU3-swiftly-api-reference)
-export SWIFTLY_AUTHORIZATION_KEY=
+# export SWIFTLY_AUTHORIZATION_KEY=
```

Comment:
Would you mind moving the variables that are now covered in 1password to a new section separate from `Required` and `Optional`, maybe `Imported from `1Password`?

Reply:
Love it! I still like the idea of having everything categorized into `Required` and `Optional` sections though, what do you think of this change with a subsection within each section for 1password variables?

[`e48cb34` (#2525)](https://github.com/mbta/skate/pull/2525/commits/e48cb348d7780664a1c2bc30992317a24be26cc0)